<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>853</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="background">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(34, 40, 49);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QFrame" name="login_area">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>450</width>
           <height>600</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
border-radius: 10px;
background-color: rgb(57, 62, 70);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLineEdit" name="lineEdit">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>210</y>
            <width>251</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Swis721 Cn BT</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	border: 2px solid rgb(221, 221, 221);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30, 40, 49);
	color: rgb(166, 166, 166);
}

QLineEdit:hover{
	border: 2px solid rgb(0, 173, 181);
	
}
QLineEdit:focus{
	border: 2px solid rgb(0, 173, 181);	
	color: rgb(0, 173, 181);
}</string>
          </property>
          <property name="maxLength">
           <number>32</number>
          </property>
          <property name="placeholderText">
           <string>USER</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>270</y>
            <width>251</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Swis721 Cn BT</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	border: 2px solid rgb(221, 221, 221);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30, 40, 49);
	color: rgb(166, 166, 166);
}

QLineEdit:hover{
	border: 2px solid rgb(0, 173, 181);
	
}
QLineEdit:focus{
	border: 2px solid rgb(0, 173, 181);	
	color: rgb(0, 173, 181);
}</string>
          </property>
          <property name="maxLength">
           <number>15</number>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>PASSWORD</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>330</y>
            <width>251</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Swis721 Cn BT</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator{
	border: 3px solid rgb(0, 173, 181);
	width: 15px;
	height: 15px;
	border-radius: 10px;
	background-color: rgb(30, 40, 49);
}

QCheckBox::indicator:checked{
	border: 3px solid rgb(0, 173, 181);
	border-radius: 10px;
	background-color: rgb(25, 193, 200);
}


</string>
          </property>
          <property name="text">
           <string>SAVE USER</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>380</y>
            <width>250</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Swis721 Cn BT</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: rgb(34, 40, 49);
	border: 2px solid rgb(57, 62, 70);
	border-radius: 5px;
}

QPushButton:hover{
	background-color: rgb(0, 173, 181);
	border-radius: 5px;
}

QPushButton:pressed{
	background-color: rgb(30, 200, 190);
	border-radius: 5px;
}</string>
          </property>
          <property name="text">
           <string>LOGIN</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>510</y>
            <width>151</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Swis721 Cn BT</family>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>DON'T HAVE AN ACCOUNT?</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_3">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>520</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Swis721 Cn BT</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: rgb(34, 40, 49);
	border: 2px solid rgb(57, 62, 70);
	border-radius: 5px;
}

QPushButton:hover{
	background-color: rgb(0, 173, 181);
	border-radius: 5px;
}

QPushButton:pressed{
	background-color: rgb(30, 200, 190);
	border-radius: 5px;
}</string>
          </property>
          <property name="text">
           <string>SIGN UP</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>104</x>
            <y>459</y>
            <width>241</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Swis721 Cn BT</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(166, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>40</y>
            <width>161</width>
            <height>141</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>logo.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
